### ISSUE TITLE
FileConverter fails for symlinked raw input

### ISSUE THREAD

**Issue by bernt-matthias** [OpenMS/OpenMS#7439]
For instance this test https://github.com/OpenMS/OpenMS/blob/17a07f8bdf78e471a26e7869a33004fd046ec510/src/tests/topp/THIRDPARTY/third_party_tests.cmake#L250 if the input is a link.

Consequence is that for the Galaxy tools we can't use symlinks for the input data.

**Comment by jpfeuffer** [comment]
This is a problem with ThermoRawFileParser: https://github.com/compomics/ThermoRawFileParser/issues/46

**Comment by jpfeuffer** [comment]
https://github.com/compomics/ThermoRawFileParser/issues/108

**Comment by bernt-matthias** [comment]
Thanks for the link. Seems that this should be solved in recent versions https://github.com/compomics/ThermoRawFileParser/issues/146

**Comment by jpfeuffer** [comment]
Yep. Not sure if we include the newest version in our conda thirdparty.

Unfortunately we are having major issues updating our thirdparty due to size limitations on mulled container tests. So, we will not be able to update them in the foreseeable future and might need to drop the thirdparty package.

**Comment by bernt-matthias** [comment]
Yep. Seems to be still a problem with the latest version https://github.com/compomics/ThermoRawFileParser/issues/180

**Comment by lvclark** [comment]
Having the same issue, using the container at https://depot.galaxyproject.org/singularity/thermorawfileparser:1.4.4--ha8f3691_0. This makes it slightly inconvenient to use the tool in Nextflow (solved by setting `stageInMode 'link'` in the process definition).

**Comment by bernt-matthias** [comment]
@lvclark also see the upstream issue https://github.com/thermofisherlsms/RawFileReader/issues/11 (and maybe comment there).
